type-c-service/src/service/ucsi.rs

```diff
@@ -165,9 +235,46 @@ impl<'a> Service<'a> {
                     response.notify_opm = false;
                     response.cci = Cci::default();
                     response.data = Ok(None);
+                    self.set_cci_connector_change(state, &mut response.cci);
                     return response;
                 }
             }
         }
     }
+
+    pub(super) async fn process_ucsi_event(&self, port_id: GlobalPortId, port_event: PortStatusChanged) {
```

Comment:
This is named "process **UCSI** event" but is called and takes an argument for **port status changed** events. Where's the misnomer coming from?